### PR TITLE
github: Improve Readme and add better issue templates

### PR DESCRIPTION
**Pull request by FearlessTobi**
Both the changes to the Readme and the issue templates have been taken pretty much directly from yuzu.

The Readme should have now have a much more appealing design and more up-to-date information.
The issue templates now require the users to clearly describe their issue and to upload a log file.

Please check everything for possible copy-paste errors or other inconsistencies.

- - -
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/6276)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
+  <br>
+  <a href="https://citra-emu.org/"><img src="https://raw.githubusercontent.com/citra-emu/citra-assets/master/Main/citra_logo.svg" alt="Citra" width="200"></a>
+  <br>
+  <b>Citra</b>
+  <br>
+</h1>
+
+<h4 align="center"><b>Citra</b> is the world's most popular, open-source, Nintendo 3DS emulator.
+<br>
+It is written in C++ with portability in mind and builds actively maintained for Windows, Linux and macOS.
```
Comment (SleepingSnakezzz) [inline README.md:11]:
```suggestion
It is written in C++ with portability in mind and builds are actively maintained for Windows, Linux, Android and macOS.
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
+</p>
+
+
+## Compatibility
+
+The emulator is capable of running most commercial games at full speed, provided you meet the necessary hardware requirements.
+
+For a full list of games Citra supports, please visit our [Compatibility page](https://citra-emu.org/game/)
+
+Check out our [website](https://citra-emu.org/) for the latest news on exciting features, monthly progress reports, and more!
```
Comment (SleepingSnakezzz) [inline README.md:41]: Monthly progress reports are more of a yuzu things 😅

**Comment by SleepingSnakezzz** [comment]
I suck at Github lol

File: README.md [hunk 3]
```diff
@@ -28,30 +53,45 @@ The [Canary](https://github.com/citra-emu/citra-canary) build is based on the ma
...
 
-If you want to contribute to the user interface translation, please check out the [citra project on transifex](https://www.transifex.com/citra/citra). We centralize the translation work there, and periodically upstream translations.
+If you want to contribute to the user interface translation, please check out the [Citra project on transifex](https://www.transifex.com/citra/citra). We centralize the translation work there, and periodically upstream translations.
 
-### Building
+## Building
 
 * __Windows__: [Windows Build](https://github.com/citra-emu/citra/wiki/Building-For-Windows)
 * __Linux__: [Linux Build](https://github.com/citra-emu/citra/wiki/Building-For-Linux)
 * __macOS__: [macOS Build](https://github.com/citra-emu/citra/wiki/Building-for-macOS)
```
Comment (SachinVin) [inline README.md:74]: While you are at it, could you add the link to android as well.

**Comment by GPUCode** [comment]
Should we consider adding some screenshots too? These always make the README feel more lively

**Comment by SleepingSnakezzz** [comment]
Screenshots of what though? Any suggestions?

**Comment by GPUCode** [comment]
Hm maybe add the games like the showcase on the citra website

**Comment by FearlessTobi** [comment]
I personally think screenshots are better suited for the website.
Most projects I know only have a simple logo in their Github Readme.

File: README.md (hunk 3 above)
Comment (SachinVin) [reply README.md:75]: Thanks

**Comment by FearlessTobi** [comment]
will merge soon if no more feedback.

**Pull request merged**